### PR TITLE
move the nav controls to left in full screen mode - in macOS Fix #7872

### LINKED ISSUE #7872: When fullscreen (on macOS), move the nav controls left
Screenshot of user fullscreen:
image

Notice in the top left there is a blank area where the traffic lights used to be (red / yellow / green buttons).  This area should disappear

### PR DESCRIPTION
**Pull request by kumarrishav**
- [ ] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Resolves #7872

Test Plan:
screen shot 2017-04-27 at 12 12 15 am

### CODE REVIEW

**Comment by luixxiul** [comment]
Please look for something like `:not(.fullscreen)`.

**Comment by kumarrishav** [comment]
@luixxiul How should i check, if screen is in full screen mode or not?

**Comment by NejcZdovc** [comment]
@kumarrishav so instead of modifying less file, you need to modify some js file. More specifically `app/renderer/components/navigation/navigator.js`. Here you need to check if window is maximized and if so you need to adjust `margin-left` for div with class `backforward`. Here you can see our [state file](https://github.com/brave/browser-laptop/blob/master/docs/state.md). Checkout `isMaximized` in `ui` section.

**Comment by kumarrishav** [comment]
@Liunkae  review? 
screen shot 2017-04-28 at 4 48 42 pm
screen shot 2017-04-28 at 4 48 51 pm

File: app/renderer/components/navigation/navigator.js [hunk 1]
```diff
@@ -225,7 +225,7 @@ class Navigator extends ImmutableComponent {
           onDragOver={this.onDragOver}
           onDrop={this.onDrop}
         >
-          <div className='backforward'>
+          <div className={'backforward' + (isFullScreen() ? ' fullscreen' : '')}>
```
Comment (NejcZdovc) [inline app/renderer/components/navigation/navigator.js]: Please use `cx` syntax for class name, you can see example bellow. Thank you

Comment (kumarrishav) [inline app/renderer/components/navigation/navigator.js]: fixed.  I wasn't aware of this utility func. @NejcZdovc Thanks .

**Pull request merged**